en/docs/get-started/quick-start-guide.md

```diff
@@ -4,17 +4,44 @@ Let's get started with WSO2 Micro Integrator by running a simple integration use
 
-3. Go to the [WSO2 Micro Integrator web page](https://wso2.com/integration/micro-integrator/#), click **Download**, and then click **Zip Archive** under the 'Latest Release' section to download the Micro Integrator distribution as a ZIP file.
-4. Extract the ZIP file. The extracted folder will be referred as the `<MI_HOME>` folder.
-5. Install WSO2 Micro Integrator [Visual Studio Code extension]({{base_path}}/develop/mi-for-vscode/install-wso2-mi-for-vscode/) (MI for VS Code)
-6. Download the [sample files]({{base_path}}/assets/attachments/quick-start-guide/mi-qsg-home.zip). From this point onwards, let's refer to this directory as `<MI_QSG_HOME>`.
-7. Download [curl](https://curl.haxx.se/) or a similar tool that can call an HTTP endpoint.
+- **Apache Maven:** Ensure Apache Maven is installed and its path is correctly set within the system's PATH environment variable.
+- **WSO2 Micro Integrator 4.3.0 Runtime:** Set up WSO2 Micro Integrator 4.3.0 runtime on your machine.
+    1. Download the Micro Integrator 4.3.0 distribution as a ZIP file from [here](https://github.com/wso2/micro-integrator/releases/download/v4.3.0/wso2mi-4.3.0.zip).
+    2. Extract the ZIP file. Hereafter, this extracted folder will be referred to as the `<MI_HOME>` folder.
+
+After completing the steps above, follow the instructions below to set up the workspace:
+
+1. Launch VS Code with the Micro Integrator extension installed.
+
+2. Click on the Micro Integrator icon on the Activity Bar of the VS Code editor.
+
+    <a href="{{base_path}}/assets/img/develop/mi-for-vscode/mi-vscode-extension.png"><img src="{{base_path}}/assets/img/develop/mi-for-vscode/mi-vscode-extension.png" alt="Mi VS Code Extension" width="80%"></a>
+
+3. Click on the **Command Palette** on the top of the VS Code.
+
+    <a href="{{base_path}}/assets/img/develop/mi-for-vscode/qsg/command-palette.png"><img src="{{base_path}}/assets/img/develop/mi-for-vscode/qsg/command-palette.png" alt="Command palette" width="70%"></a>
+
+4. Type `>` to show the available commands. Alternatively, you can open the command palette in VS Code by entering `Command`+`Shift`+`P` on macOS and `Ctrl`+`Shift`+`P` on Windows.
+
+5. Select **MI: Add MI server** from the list of available commands.
+
+6. Click **Add MI server** to add a Micro Integrator server.
+
+7. Select the folder where `<MI_HOME>` is located. This wll be set as the **current server path**.
+
+    <a href="{{base_path}}/assets/img/develop/mi-for-vscode/qsg/current-server-path.png"><img src="{{base_path}}/assets/img/develop/mi-for-vscode/qsg/current-server-path.png" alt="Current server path" width="50%"></a>
+
+8. Download the [sample files]({{base_path}}/assets/attachments/quick-start-guide/mi-qsg-home.zip). From this point onwards, let's refer to this directory as `<MI_QSG_HOME>`.
+
+9. Download [curl](https://curl.haxx.se/) or a similar tool that can call an HTTP endpoint.
 
 ## What you'll build
 
```

Comment:
The Healthcare API calls the Pine Valley Hospital back-end service and gets the queried information.
->

The Healthcare API calls the Pine Valley Hospital back-end service to retrieve requested information.

Reply:
The Healthcare API calls the Grand Oak Hospital back-end service and gets the queried information.
->

The Healthcare API calls the Grand Oak Hospital back-end service to retrieve requested information.

Reply:
The Grand Oak service accept GET requests in the following service endpoint URL
->

The Grand Oak service accepts GET requests in the following service endpoint URL